tools/watch_logs.py

```diff
@@ -10,6 +10,44 @@
 import time
 import threading
 from baselayer.log import colorize
+import logging
+import socket
+from logging.handlers import SysLogHandler
+
+from app.env import load_env
+
+env, cfg = load_env()
+
+
+# check 3rd party logging, if enabled, and make sure that
+# it is set up properly
+if cfg.get('external_logging'):
```

Comment:
Suggestion: validate the `external_logging` attribute using JSONschema.

Reply:
I'd rather hold off on this until we have multiple services and understand what a common schema could look like. Ok?